app/migration/builders/ect/school_periods.rb

```diff
@@ -14,6 +14,10 @@ def build
           school = School.find_by!(urn: period.urn)
           ::ECTAtSchoolPeriod.create!(teacher:,
                                       school:,
+                                      appropriate_body_type: period.appropriate_body_type,
+                                      programme_type: period.programme_type,
+                                      appropriate_body_id: period.appropriate_body_id,
+                                      lead_provider_id: period.lead_provider_id,
```

Comment:
It's my understanding that `appropriate_body_type`, `lead_provider_id` and `appropriate_body_id` are just temporarily present on `ECTAtSchoolPeriod` and  in the longer term they'll shift to `InductionPeriod` and `TrainingPeriod` respectively.

Maybe we can put them together here and leave a comment, just to inform other people reading it that it's not going to be there forever.

Reply:
I'm wondering if it's better to add a note in the classes where the temp attributes are defined. 

I added one in the `Migration::SchoolPeriod` class for now.